src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$day-size: 100px;
```

Comment:
Consider using a more descriptive variable name for the day size, like $day-box-size or $day-cell-size.